### PR TITLE
OpenSearch Migration

### PR DESCRIPTION
**Pull request by ambrozic**
### Description of change

Migrating from ElasticSearch to OpenSearch

### CODE REVIEW

**Review by tomkingston — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -35,8 +32,8 @@ job_defaults: &job_defaults
     DJANGO_SETTINGS_MODULE: config.settings.local
     ES_INDEX_PREFIX: test_index
     ES5_URL: http://localhost:9200
-    # This has been readded to allow tests utilizing elasticsearch to be unskipped
-    ELASTICSEARCH_URL: http://localhost:9200
+    # This has been re-added to allow tests utilizing OpenSearch to be un-skipped
```
Comment (craigjbass) [inline .circleci/config.yml:35]:
```suggestion
```

I think this comment is out of date.

File: .circleci/config.yml [hunk 2]
```diff
@@ -48,7 +45,7 @@ job_defaults: &job_defaults
     DIT_EMAIL_DOMAINS: trade.gov.uk,digital.trade.gov.uk
     DATA_HUB_FRONTEND_ACCESS_KEY_ID: frontend-key-id
     DATA_HUB_FRONTEND_SECRET_ACCESS_KEY: frontend-key
-    ES_APM_ENABLED: 'True'
+    ES_APM_ENABLED: 'False'
```
Comment (craigjbass) [inline .circleci/config.yml:48]: Is ES_APM_ENABLED set to true in dev/staging or uat?

File: config/settings/common.py [hunk 3]
```diff
@@ -333,6 +333,11 @@
 else:
     ES_URL = env('ES5_URL')
 
+if 'opensearch' in VCAP_SERVICES:
+    OPENSEARCH_URL = VCAP_SERVICES['elasticsearch'][0]['credentials']['uri']
```
Comment (craigjbass) [inline config/settings/common.py:337]: Part of me does wonder whether we should just supply VCAP_SERVICES locally and reduce differences between prod and local.

**Review by craigjbass — APPROVED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (ambrozic) [reply .circleci/config.yml:48]: yes, both are set to `True`

**Review by elcct — APPROVED** [review]

**Pull request merged**